rafiki/model/dev.py

```diff
@@ -228,7 +228,7 @@ def make_predictions(queries: List[Any], task: str, py_model_class: Type[BaseMod
         out_prediction = ensemble_method([prediction])
         out_predictions.append(out_prediction)
 
-    print('Predictions: {}'.format(out_predictions))
+    # print('Predictions: {}'.format(out_predictions))
```

Comment:
why do you comment out this line?

Reply:
Sorry, I forget to revoke the comments (in dev.py and utils.py) before committing. 
Comment out this line is to avoid terminal stuck when testing a large number of queries.